.github/workflows/test-linux.yml

```
@@ -26,9 +26,10 @@ on:
 jobs:
   Test:
     name: ${{ matrix.instance }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-2004
```

```suggestion
    runs-on: ubuntu-20.04
```
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources